responses/__init__.py

```diff
@@ -668,6 +668,9 @@ def __exit__(self, type, value, traceback):
     def activate(self, func):
         return get_wrapped(func, self)
 
+    def calls_for_match(self, match):
```

Comment:
```suggestion
    def _calls_for_match(self, match):
```

I'd rather we not have this as a public method.